code/android-core-library/src/phone/java/com/adobe/marketing/mobile/MobileCore.java

```diff
@@ -165,7 +165,7 @@ public static void setLogLevel(@NonNull LoggingMode mode) {
      */
     @NonNull
     public static LoggingMode getLogLevel() {
-        return Log.getLogLevel();
+        return com.adobe.marketing.mobile.services.Log.getLogLevel();
```

Comment:
@praveek  we can clean up the core extension after the legacy classes are moved to a separate artifact.( then we won't see 2 Log classes in core extension)